<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWifi</class>
 <widget class="QMainWindow" name="SettingWifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingWifi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="InputWidget" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QLabel" name="SSID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ssid:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SSIDInput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>41</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordInput">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>101</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Password">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>69</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cancel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="LastPage">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>last page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextPage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>next page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TestText">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Test">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Network connection mode: </string>
    </property>
    <widget class="QRadioButton" name="wifiButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Wifi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="SIMButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SIM Card</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing WIFI SSID and password: </string>
    </property>
    <widget class="QPushButton" name="AddSSID">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add 
SSID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clean">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delele
All</string>
     </property>
    </widget>
    <widget class="QListWidget" name="SSIDList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SSID List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteSSID">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete
 SSID</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="SSIDContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveSSID">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>310</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Timezone</string>
    </property>
    <widget class="QComboBox" name="TimeZoneCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Clock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Software Update</string>
    </property>
    <widget class="QPushButton" name="Update">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="UpdateLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(140, 140, 140);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing SIM APN</string>
    </property>
    <widget class="QPushButton" name="AddAPN">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add 
APN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanAPN">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delele 
All</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="APNContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="saveAPN">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteAPN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete 
APN</string>
     </property>
    </widget>
    <widget class="QListWidget" name="APNList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="APNLabel">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>APN List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>231</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Charge Log</string>
    </property>
    <widget class="QTextEdit" name="LogContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteLOG">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>430</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QListWidget" name="LogList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Log List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="TestBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TestBox2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <zorder>LastPage</zorder>
   <zorder>NextPage</zorder>
   <zorder>TestText</zorder>
   <zorder>Test</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>TestBox</zorder>
   <zorder>TestBox2</zorder>
   <zorder>InputWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
